index.d.ts

```diff
@@ -2,7 +2,20 @@ declare module "typy" {
 
 	type SafeFunction = (...args: any[]) => any;
 
-	export default function typy (input: any, objectPath?: string): Typy
+	export function t (obj: any, nestedKeys?: string): Typy;
```

Comment:
One last change, you forgot to export `t` as default. It needs to be declared as named and default export since that's how the code is currently structured right now.
```js 
export default t;
```

As it is right now, it will cause a TS error without it since there's no function signature found for it if you try to import it as such: import t from 'typy'

Reply:
fixed